sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -930,7 +947,7 @@ def __init__(
         :rtype: ~azure.storage.blob.lease.LeaseClient
```

Comment:
I'd use this PR to update rtypes here to something like this
`~azure.storage.blob.aio.lease_async.LeaseClient`

Reply:
Good point, I'll do this.